### PR TITLE
SOV-4072: dollar value issue

### PR DESCRIPTION
**Pull request by rick23p**
https://sovryn.atlassian.net/browse/SOV-4072
https://sovryn.atlassian.net/browse/SOV-4065

### CODE REVIEW

File: apps/frontend/src/app/5_pages/MarketMakingPage/components/AmbientMarketMaking/components/AmbientPoolPositions/components/AmbientPositionPrices/AmbientPositionPrices.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const AmbientPositionPrices: FC<AmbientPositionPricesProps> = ({
   position,
   pool,
 }) => {
-  if (position.positionType === 'ambient') {
+  if (position.bidTick === MIN_TICK && position.askTick === MAX_TICK) {
```
Comment (creed-victor) [inline apps/frontend/src/app/5_pages/MarketMakingPage/components/AmbientMarketMaking/components/AmbientPoolPositions/components/AmbientPositionPrices/AmbientPositionPrices.tsx]:
```suggestion
  if (position.positionType === 'ambient' || (position.bidTick === MIN_TICK && position.askTick === MAX_TICK)) {
```

File: apps/frontend/src/hooks/useDollarValue.ts [hunk 2]
```diff
@@ -20,17 +20,29 @@ export function useDollarValue(
...
   const { price: btcPrice } = useGetRBTCPrice();
 
+  const currentChainId = useCurrentChain();
+  const smartRouter = useMemo(
+    () => new SmartRouter(getProvider(currentChainId), SWAP_ROUTES),
+    [currentChainId],
+  );
+
   const { value: usdPrice, loading } = useCacheCall(
     `dollarValue/${asset}`,
```
Comment (creed-victor) [inline apps/frontend/src/hooks/useDollarValue.ts]:
```suggestion
    `dollarValue/${chainId || chain}/${asset}`,
```

File: apps/frontend/src/hooks/useDollarValue.ts [hunk 3]
```diff
@@ -20,17 +20,29 @@ export function useDollarValue(
...
+    }
+  } else if (asset.toLocaleLowerCase() === 'weth') {
+    asset = COMMON_SYMBOLS.ETH;
+  }
   const assetDetails = useTokenDetailsByAsset(asset, chainId || chain);
   const dllrDetails = useTokenDetailsByAsset(
     COMMON_SYMBOLS.DLLR, // todo: define USD equivalent token for all chains in config
     chainId || chain,
   );
   const { price: btcPrice } = useGetRBTCPrice();
```
Comment (creed-victor) [inline apps/frontend/src/hooks/useDollarValue.ts]: Remove it altogether
```suggestion
```

File: apps/frontend/src/hooks/useDollarValue.ts [hunk 4]
```diff
@@ -74,18 +87,17 @@ export function useDollarValue(
...
     } else {
       return decimalic(weiAmount)
-        .mul(assetDetails?.isNative ? btcPrice : usdPrice)
+        .mul(
+          [COMMON_SYMBOLS.BTC, COMMON_SYMBOLS.WBTC].includes(
+            asset.toUpperCase(),
+          )
+            ? btcPrice
+            : usdPrice,
+        )
```
Comment (creed-victor) [inline apps/frontend/src/hooks/useDollarValue.ts]:
```suggestion
        .mul(usdPrice)
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/5_pages/MarketMakingPage/components/AmbientMarketMaking/components/AmbientPoolPositions/components/AmbientPositionPrices/AmbientPositionPrices.tsx [hunk 5]
```diff
@@ -16,8 +16,10 @@ export const AmbientPositionPrices: FC<AmbientPositionPricesProps> = ({
   pool,
 }) => {
   const isAmbient = useMemo(
-    () => position.positionType === 'ambient',
-    [position.positionType],
+    () =>
+      position.positionType === 'ambient' ||
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/5_pages/MarketMakingPage/components/AmbientMarketMaking/components/AmbientPoolPositions/components/AmbientPositionPrices/AmbientPositionPrices.tsx]:
```suggestion
      position.positionType === PoolPositionType.ambient ||
```

**Review by creed-victor — APPROVED** [review]

**Pull request merged**